api-tests/core/content-manager/fields/uid.test.api.js

```diff
@@ -234,7 +234,7 @@ describe('Test type UID', () => {
       expect(conflicting.statusCode).toBe(400);
     });
 
-    // TODO: Fix uniqueness and validations in document service
+    // TODO: Fix uniqueness and validations in document service -
     test.skip('Cannot set value to be null', async () => {
```

Comment:
Could you take a look at this one when you have the time @jhoward1994  It looks like uid fields can be set to null

Reply:
https://github.com/strapi/strapi/pull/19448

we can discuss this in there if you like?

Reply:
yess! completely missed that PR

https://github.com/strapi/strapi/pull/19427 We can take a look at how to integrate  your work into this PR